packages/zoe/src/zoeService/startInstance.js

```diff
@@ -43,7 +43,7 @@ export const makeStartInstance = (
     );
     // AWAIT ///
 
-    const { adminNode, root } = await zoeInstanceStorageManager.createZCFVat();
+    const { adminNode, root } = await zoeInstanceStorageManager;
```

Comment:
```suggestion
    const { adminNode, root } = zoeInstanceStorageManager;
```

Reply:
An `await` ripe for removal! Thanks for catching this.